client/signup/steps/theme-selection/index.jsx

```diff
@@ -126,9 +123,9 @@ const ThemeSelectionStep = React.createClass( {
 				</div>
 				<div className={ themesWrapperClassName } >
 					<StepWrapper
-						fallbackHeaderText={ this.translate( 'Choose a theme.' ) }
-						fallbackSubHeaderText={ this.translate( 'No need to overthink it. You can always switch to a different theme later.' ) }
-						subHeaderText={ this.translate( 'Choose a theme. You can always switch to a different theme later.' ) }
+						fallbackHeaderText={ translate( 'Choose a theme.' ) }
+						fallbackSubHeaderText={ translate( 'No need to overthink it. You can always switch to a different theme later.' ) }
```

Comment:
Seems a strange case. The line is exactly 140 characters long, according to PHPStorm and the linter here doesn't throw an error :) 

Updated it anyway to split it in multiple lines, to shorten the line.